packages/node/@types/zeit__ncc/index.d.ts

```diff
@@ -1,7 +1,7 @@
 declare function ncc(
   entrypoint: string,
   options?: ncc.NccOptions
-): ncc.NccResult;
```

Comment:
The type definition in this file only defines the return type where `watch` is `true`, it seems.

I'd like to augment this to have the proper return type based on the presence of `watch: true`, but I couldn't figure out how to add that specific type definition overload. Any suggestions?

Reply:
Are we planning to use `watch: true`? If not, I wouldn't worry about it